migrations/0006_create_tags_tables.sql

```diff
@@ -0,0 +1,25 @@
+-- Migration number: 0006 	 2025-08-28T22:51:05.213Z
+
+-- Tags table
+CREATE TABLE tags (
+  id TEXT PRIMARY KEY,
+  name TEXT NOT NULL UNIQUE,
+  color TEXT NOT NULL DEFAULT 'neutral',
+  created_at TEXT NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TEXT NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
```

Comment:
This means that tags are global to the whole system. Anyone can create tags. I feel like we can't enforce that the name is unique... unless we're enforcing that the name is unique _for that one user_.